test/integration/services/mongoose/account-api-keys.spec.ts

```diff
@@ -0,0 +1,106 @@
+import { hashApiKey, randomApiKey } from "@domain/accounts"
+import { CouldNotFindError } from "@domain/errors"
+import { AccountApiKeysRepository } from "@services/mongoose"
+
+const repo = AccountApiKeysRepository()
+const delta = 30 // days
```

Comment:
how about making the comment part of the variable name: `expiration_in_days` or `delta_days` or something like hat.

Reply:
fixed